lib/e2e.test.ts

```diff
@@ -47,7 +47,7 @@ class TestHandler {
   public read(
     @Header('Content-Type') contentType: string,
     @Query('id') id: string,
-    @Query('step', ParseNumberPipe) step: number,
+    @Query('step', ParseNumberPipe({ nullable: false })) step: number,
```

Comment:
Can we add an optional `label` parameter to the options, so we can override the `name` in the error message?

Reply:
Why would we want to override it when we use the actual query parameter name that is in the url itself?

Reply:
If it's something like `firstName` you might want to show it to the user as `First name`. But since it's for query params and probably not used for things like form submissions, it doesn't matter that much if it's written in a dev-friendly way. (Will mean we expect FE to handle the non-dev friendly validation errors with things like form validation).

Reply:
I say let's skip it for now and make it a separate issue.

Reply:
👍